openrtb_ext/bidders.go

```diff
@@ -118,6 +118,7 @@ const (
 	BidderEngageBDR         BidderName = "engagebdr"
 	BidderEPlanning         BidderName = "eplanning"
 	BidderEpom              BidderName = "epom"
+	BidderEVolution         BidderName = "e_volution"
```

Comment:
The bidder name `"e_volution"` is fine here.

Reply:
I tried changing the name to "evolution" everywhere. Only in this place left "e_volution". As a result, my tests broke. What did I do wrong ? `static/bidder-params` and `static/bidder-info` in this case should have what name?

Reply:
@SyntaxNode

Reply:
@e-volution-tech Can you please tell us more about what tests failed and what were the failures so that we can better help you?

Reply:
And to your second question, you will have to keep the files in `static/bidder-params` and `static/bidder-info` as `e_volution.json` and `e_volution.yaml` as the code tries to look for those files based on the bidder name provided here in `openrtb_ext/bidders.go`

Reply:
@mansinahar
I understood what the problem is. I just didn't use the name correctly everywhere. Everything is OK now. Thank you so much.